infrastructure/prod.tfvars

```diff
@@ -1,7 +1,13 @@
 vault_section = "prod"
 capacity = "2"
+reupload_delay = "10000"
+reupload_enabled = true
+scan_delay = "10000"
+scan_enabled = true
+
+orchestrator_notifications_task_enabled = "true"
+
 api_gateway_test_valid_certificate_thumbprint = "3B0D23863EE7EF90290AD02B0ACFD7C37ED5B330"
 allowed_client_certificate_thumbprints = ["919F8A770C2A2F06933BA11551001ED0581B8848"]
 
 blob_processing_delay_in_minutes = "30"
-error_notifications_enabled = "false"
```

Comment:
are credentials and url updated for/in production vault?

Reply:
yes done that